### ISSUE TITLE
list change detection doesn't take commit message changes into account

### ISSUE THREAD

**Issue by drewdeponte** [uptech/git-ps-rs#75]
If you change a patch description it should recognize that as a patch difference and let you know so you can sync, etc. it.

**Comment by drewdeponte** [comment]
I don't think there is anything native to Git that can help us with this.

I think we tould have to create our own representational format of the patch in the stack that we could hash, then do the same for the patch in any branches, and then compare the hashes to see if they match or they don't.

So doable but not as simple as getting some hash from Git.

**Comment by Alizter** [comment]
Something simple like hashing a normalized version of the patch file should be possible. Although there are probably better solutions.

**Comment by drewdeponte** [comment]
Yeah, currently we are using the diff hash of the commit. This is great. It just doesn't include changes to the description, summary, commiter, author, or signatures.

So we need to create a standardized format that we can has that will include those things but not include the commiter timestamp.

Then we can just hash it and compare the hashes to determine change.
